lib/elixir/pages/library-guidelines.md

```diff
@@ -200,7 +200,31 @@ The code above says we are only bringing in the functions from `MyLib` so we can
 
 If the module you want to invoke a function on has a long name, such as `SomeLibrary.Namespace.MyLib`, and you find it verbose, you can leverage the `alias/2` special form and still refer to the module as `MyLib`.
 
-While there are situations where `use SomeModule` is necessary, `use` should be skipped when all it does is to `import` or `alias` other modules. In a nutshell, `alias` should be preferred, as it is simpler and clearer than `import`, while `import` is simpler and clearer than `use`.
+### Avoid undocumented `use SomeModule`
+
+In some situations, where you need to do more than importing and aliasing modules, allowing a developer to `use SomeModule` may be necessary. The benefit of `use SomeModule` is that it provides a common extension point for the Elixir ecosystem. However, given `use SomeModule` can execute any code, it may not be easy for developers to understand the impact of `use SomeModule`.
+
+For this reason, to provide guidance and clarity, we recommend developers to include an admonition block in their `@moduledoc` that explains how `use SomeModule` impacts their code. As an example, the `GenServer` documentation outlines:
```

Comment:
I struggled a bit when reading this one personally, and I think the reason if because we use "developers" both for authors of the used module and for users using it. I know that both are probably developers but it might read more naturally with a consistent terminology for each side?

Reply:
```suggestion
In some situations, where you need to do more than importing and aliasing modules, allowing a developer to `use SomeModule` may be necessary. The benefit of `use SomeModule` is that it provides a common extension point for the Elixir ecosystem. However, given `use SomeModule` can execute any code, it may not be easy for developers to understand the impact of `use SomeModule`.

For this reason, to provide guidance and clarity, we recommend library authors to include an admonition block in their `@moduledoc` that explains how `use SomeModule` impacts the developer code. As an example, the `GenServer` documentation outlines:
```

Reply:
Not sure how to suggest over a suggestion, but there are a few grammatical changes here:

> For this reason, to provide guidance and clarity, we recommend library authors include an admonition block in their `@moduledoc` that explains how `use SomeModule` impacts the developer's code. As an example, the `GenServer` documentation outlines:

(Remove a `to` and make it `developer's`)